### PR TITLE
feat: Make login a shared method

### PR DESCRIPTION
**Pull request by joostlek**
Make login a shared method

### CODE REVIEW

**Comment by chemelli74** [comment]
Checking from mobile, so I may have missed it but I cannot see the different payload values for the 2 host types

**Comment by joostlek** [comment]
They are implemented in the subclasses

**Comment by chemelli74** [comment]
> They are implemented in the subclasses

Still cannot see the payload for VEDO in your PR.

**Comment by joostlek** [comment]
afbeelding

**Comment by chemelli74** [comment]
> afbeelding
> 

Added a comment, you are not pointing to the missing payload with this screenshot

File: src/aiocomelit/api.py [hunk 1]
```diff
@@ -160,11 +162,16 @@ async def _check_logged_in(self, host_type: str) -> bool:
 
         return logged
 
-    async def _login(self, payload: dict[str, Any], host_type: str) -> bool:
+    async def login(self) -> bool:
+        """Login to Serial Bridge device."""
+        payload = {"dom": self.device_pin}
+        return await self._login(payload)
```
Comment (joostlek) [inline src/aiocomelit/api.py:168]: Here?

Comment (chemelli74) [reply src/aiocomelit/api.py:168]: For the serial bridge, but for VEDO ?

Comment (joostlek) [reply src/aiocomelit/api.py:168]: This is for both

Comment (chemelli74) [reply src/aiocomelit/api.py:168]: > This is for both

Which is wrong, as the 2 devices have different payload.
Recheck current code ;-)

Comment (joostlek) [reply src/aiocomelit/api.py:168]: Oh lol, I overlooked the dom/code 5-6 times

Comment (chemelli74) [reply src/aiocomelit/api.py:168]: I think we can leave library as it is in the end

Comment (joostlek) [reply src/aiocomelit/api.py:168]: I think it will help if the login is at least at the shared part

**Pull request merged**